postcss.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  plugins: [
+    require('autoprefixer')({ // eslint-disable-line
+
+    }),
+    require('./stack/tasks/postcss-font-base64.js')({ // eslint-disable-line
```

Comment:
I would provide it as a separate module in the future. For now is okay.

Reply:
i though that as well. Totally agree